src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todo: Todo;
+  handleDeleteTodo: (todoId: number) => void;
+  loading?: boolean;
+};
+
+const TodoItem: React.FC<Props> = React.memo(({ todo, handleDeleteTodo, loading = false }) => {
+
+  const [deleteId, setDeleteId] = useState<number | null>(null);
+
+  return (
+    <div data-cy="Todo" className={cn("todo", { "completed": todo.completed })}>
+      <label className="todo__status-label" htmlFor={`todo-${todo.id}`}>
```

Comment:
Fix this linter error 
![Uploading image.png…]()

Reply:
Linter issue is fixed